src/component/Header/Header.jsx

```diff
@@ -0,0 +1,311 @@
+    height: "40.111px",
+  },
+  search: {
+    color: "red",
+    width: "24%",
+    borderBottom: "none",
+  },
+  rightMenu: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-evenly",
+    width: "40%",
+    color: "var(--nav-bar-text, #7B7881)",
+  },
+  rightMenuIcon: {
+    fontSize: "14px",
+  },
+  menuIcon: {
+    display: "flex",
+    justifyContent: "space-around",
+    alignItems: "center",
+    color: "#937DC2",
+    width: "10%",
+    [theme.breakpoints.down("sm")]: {
+      width: "28%",
+    },
+    [theme.breakpoints.down("xs")]: {
+      justifyContent: "end",
+      width: "28%",
+    },
+  },
+  secondHead: {
+    display: "flex",
+    justifyContent: "space-around",
+    alignItems: "center",
+    width: "60%",
+    color: "var(--body-text, #4D4C4C)",
+  },
+  call: {
+    color: "#937DC2",
+    border: " 1px solid #937DC2",
+    marginLeft: "5%",
+  },
+  phone: {
+    color: "#937DC2",
+  },
+  seconditem: {
+    justifyContent: "space-between",
+    width: "80%",
+    color: "var(--body-text, #4D4C4C)",
+  },
+  RsecondHead: {
+    width: "30%",
+    justifyContent: "flex-end",
+    display: "flex",
+    alignItems: "center",
+  },
+  drawer: {
+    width: drawerWidth,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+  },
+  searchIcon: {
+    position: "relative",
+    right: "21px",
+  },
+}));
+const Header = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.down("sm"));
+  const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
+
+  const [drawerOpen, setDrawerOpen] = useState(false);
+
+  const handleDrawerOpen = () => {
+    setDrawerOpen(true);
+  };
+
+  const handleDrawerClose = () => {
+    setDrawerOpen(false);
+  };
+
+  return (
+    <>
+      <AppBar className={classes.menuBar}>
+        <Toolbar className={classes.menuItem}>
+          <Link to={"/"} style={{ textDecoration: "none" }}>
+            <Box className={classes.logo}>
+              <Typography className={classes.logoText}>
+                B-W
+                <Box className={classes.logoImg}>
+                  <Brightness1 className={classes.circleIcon} />
+                </Box>
+                rld
+              </Typography>
+              <ImportContactsTwoTone className={classes.book} />
+              <Box className={classes.vLine}></Box>
+              <Typography className={classes.cap}>
+                We Love <br />
+                Books{" "}
+              </Typography>
+            </Box>
+          </Link>
+          {(isMobile || isTablet) && (
+            <IconButton>
+              <Search className={classes.searchIcon} />
+            </IconButton>
+          )}
+          {!isMobile && !isTablet && <SearchBar className={classes.search} />}
+          {!isTablet && (
+            <>
+              <Box className={classes.rightMenu}>
+                <Typography className={classes.rightMenuIcon}>

```

Comment:
for these kind of stuff always defne a const and loop over it instead of harcoded items